### PR TITLE
Es6 branch

### PR DESCRIPTION
**Pull request by RoyJumah**
## In this pull request I did the following;
- I added markup and styles for awesome books
- By using the export and import functionalities I moved the scripts into the module folder
- I used the Luxon library to obtain and display the date in the UI
- I customized the Readme file and added a live demo link
- I deployed the page using Github pages

### CODE REVIEW

File: modules/Bookstore.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export default class Bookstore {
```
Comment (Rustamxon7) [inline modules/Bookstore.js:1]: - [ ]  Please change `class` based components to `function` based 😊

File: modules/UI.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import Bookstore from './Bookstore.js';
+import displayDate from './displayDate.js';
+// import displayDate from './displayDate.js';
```
Comment (Rustamxon7) [inline modules/UI.js:3]: - [ ] Please remove unused commented codes from your file :).

File: modules/UI.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+export default class UI {
```
Comment (Rustamxon7) [inline modules/UI.js:5]: - [ ] Please change `class` based components to `function` based 😊

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
Hello @RoyJumah,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/Bookstore.js (hunk 1 above)
Comment (RoyJumah) [reply modules/Bookstore.js:1]: image
 As per the project requirements, we were only supposed to refractor methods and functions to arrow functions which I have already done @Rustamxon7

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢

Hi @RoyJumah,

Good job on making the changes highlighted by the previous reviewer 👍 

Your project is complete!  

## Highlights:
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ Used ES6 and arrow function syntax 👍 
✅ Add and remove features work fine as expected 👍

## Recommendation:

- [ ] It would be nice if you could add a space after the by in the phrase Book byPerson Name to Book by Person Name. 👍 
image

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**